src/Translations/Loader/TranslationsLoader.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace Becklyn\PhpCs\Translations\Loader;
+
+use Symfony\Component\Translation\Loader\YamlFileLoader;
+use Symfony\Component\Translation\MessageCatalogue;
+use Symfony\Component\Translation\Reader\TranslationReader;
+
+class TranslationsLoader
+{
+    /**
+     * Loads all translations
+     *
+     * @param array $dirs
+     * @param array $locales
+     *
+     * @return array
+     */
+    public function loadTranslations (array $dirs, array $locales) : array
+    {
+        $reader = new TranslationReader();
+        $reader->addLoader('yaml', new YamlFileLoader());
```

Comment:
There are still some (older) projects where we're using `.xlf` files in conjunction to `.yml`. Would it cause problems as we may be referencing some translations that don't exist in our YAML catalogues?

Reply:
Yes, but that command is intended only for the newer libraries / projects.
These older projects should just be ported to the other format.